setup.py

```diff
@@ -36,8 +37,20 @@
 with open("README.md") as file:
     long_description = file.read()
 
+OMP_CONFIG = defaultdict(
+    lambda: ["-fopenmp"], {"unix": ["-fopenmp"], "msvc": ["/openmp"]}
```

Comment:
unix: this is a bit optimistic but will work for GCC and Clang at least

Reply:
furthermore: AppleClang does not support OpenMP by default.
If you want to unconditionally add an OpenMP prerequisite, you could install `brew install libomp` on CI (and need to document this new dependency on macOS).

Practically, one might want to implement a compiler check and keep OpenMP optional.